### PR TITLE
Add mui@3.4.0 w/ git auto-update

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #10514

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/dcloudio/mui
 * Official website (optional, not the repository): https://dev.dcloud.net.cn/mui
 * License and its reference: [MIT](https://github.com/dcloudio/mui/blob/v3.4.0/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 620
   - Count of stars: 5361
   - Count of forks: 2703

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by extend1994** [comment]
@cdnjs/intern3 please help me review this pull request, thank you.

**Comment by extend1994** [comment]
@PeterDaveHello please help me review this pull request, thank you.

**Comment by extend1994** [comment]
@PeterDaveHello I rewrote the description in English version. Please help me review this pull request, thank you.

**Comment by PeterDaveHello** [comment]
@hare1039 @sufuf3 please take a look first.

File: ajax/libs/mui/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "mui",
+  "filename": "js/mui.min.js",
+  "description": "A effective front-end framework clostest to the native app",
```
Comment (hare1039) [inline ajax/libs/mui/package.json]: How about
The most native-speed front-end framework

BTW, you got typo `clostest`

Comment (PeterDaveHello) [inline ajax/libs/mui/package.json]: I think everybody has his/her own presentation, so it'll be better to point out the "problem" or tell us what's the better point in yours.

**Comment by maruilian11** [comment]
I think this PR is ok, just need to rebase with our latest master.

**Comment by extend1994** [comment]
@PeterDaveHello I rebased the branch, please help me review it again, thank you.

**Comment by extend1994** [comment]
@PeterDaveHello After I request your review first time, I found something wrong and then update it again just now, so it seems I request your review before I passed the CI test. Please help me review this pull request, thank you.

File: ajax/libs/mui/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "mui",
+  "filename": "js/mui.min.js",
+  "description": "A effective front-end framework closest to the native app",
```
Comment (PeterDaveHello) [inline ajax/libs/mui/package.json]: I think there are problems here.

Should `closest` be used with `the`? The `native` here actually means its performance which is really close to the native application, but I can't see the correct meaning in the description here.

Comment (extend1994) [inline ajax/libs/mui/package.json]: @PeterDaveHello How about `A effective front-end framework whose performance is closest to the native app`?

Comment (PeterDaveHello) [inline ajax/libs/mui/package.json]: I think it's just a little bit like Taiwanese English, mind to think about how to improve it again?

Comment (extend1994) [inline ajax/libs/mui/package.json]: @PeterDaveHello Then `A effective front-end framework allowing users to design components, like they are on the mobile application, running on browser`?

Comment (PeterDaveHello) [inline ajax/libs/mui/package.json]: I don't have a good candidate yet but I think this one may be too long and not accurate enough, we may need more time on this, or maybe just write a individual description for it, not translate it.

Comment (extend1994) [inline ajax/libs/mui/package.json]: @PeterDaveHello Or maybe I can ask the author if he can write an English description?

Comment (PeterDaveHello) [inline ajax/libs/mui/package.json]: Sure thing 👍

**Comment by extend1994** [comment]
@maruilian11 @cdnjs/intern3 I modified the `description` in package.json according to the author's response(https://github.com/dcloudio/mui/issues/336#issuecomment-317380752), please help me review it, thank you.

**Comment by PeterDaveHello** [comment]
Ping @maruilian11

**Comment by extend1994** [comment]
@maruilian11  @pvnr0082t Please help me review this PR, thank you.

**Pull request merged**